tests/tests.pl

```diff
@@ -468,6 +474,7 @@ sub doViewer {
     print "  --elasticsearch <url> Set elasticsearch URL\n";
     print "  --debug               Turn on debuggin\n";
     print "  --valgrind            Use valgrind on capture\n";
+    print "  --skipgeo             Skip downloading Geo files, assume already present (useful for offline)\n";
```

Comment:
I'm confused why this change is needed, since if the files already exist it won't try and download them anyway, and the help says to assume they are present.